ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -46,6 +46,7 @@ - (void)handleMethodCall:(FlutterMethodCall *)call
         [self setupPurchases:apiKey
                    appUserID:appUserID
                 observerMode:observerMode
+                   useAmazon:NO
```

Comment:
we don't even need to have the parameter here, though, right? since it will always be `NO`, or at least, until Amazon Store for iOS is a thing

Reply:
I "Amazon Store for iOS is a thing" and I scared myself.

Reply:
yeah, no we can remove this. it's not needed